riverdriver/river_driver_interface.go

```diff
@@ -363,81 +383,89 @@ type JobSetStateIfRunningParams struct {
 	MetadataDoMerge bool
 	MetadataUpdates []byte
 	ScheduledAt     *time.Time
+	Schema          string
```

Comment:
see my other comment on this, I'm wondering if we can just get rid of this field and make it a completer level config? It can't update more than one schema at once anyway. Each of the helpers below would lose the extra arg too.

Reply:
See above, but most of this has changed, although `Schema` is still needed here because it's still used by the completer when invoking the driver.